content/applications/inventory_and_mrp/inventory/shipping/setup/sendcloud_shipping.rst

```diff
@@ -0,0 +1,193 @@
+============================================
+Setup of Sendcloud shipping services in Odoo
+============================================
+
+Sendcloud is a shipping service aggregator. Sendcloud facilitates the integration of European
+shipping carriers with Odoo. Once integrated, the user can select the shipping carrier on inventory
+operations in their Odoo database.
+
+.. note::
+   Here is `Sendcloud's documentation
+   <https://support.sendcloud.com/hc/en-us/articles/360059470491-Odoo-integration>`_ on setting up
+   the integration.
+
+.. important::
+   Odoo integration of Sendcloud does **not** work on free plans of Sendcloud.
+
+Setup in Sendcloud
+==================
+
+Create an account and activate carriers
+---------------------------------------
+
+To get started, go to `Sendcloud's platform <https://www.sendcloud.com>`_ to configure the account
+and generate the connector credentials. Log in with the Sendcloud account.
+
+If the user does not have a Sendcloud account, fill in the contact information and payment details.
+Sendcloud will ask for a :abbr:`VAT (Value-Added Tax Identification)` number or :abbr:`EORI
+(Economic Operators' Registration and Identification)` number. After completing the account setup,
+activate (or deactivate) the shipping carriers that will be used in the Odoo database.
+
+Warehouse configuration
+-----------------------
+
+Once the Sendcloud account is created, set up the :guilabel:`Warehouse Address`. Start in the
+Sendcloud account, go to :menuselection:`Settings --> Shipping --> Addresses`, and enter the
+:guilabel:`Warehouse address`.
+
+.. image:: sendcloud_shipping/settings-shipping.png
+   :align: center
+   :alt: Adding addresses in the Sendcloud settings.
+
+If the company would like Sendcloud to process returns as well, a :guilabel:`Return Address` is
+required. Under the :guilabel:`Miscellaneous section`, there is a field called :guilabel:`Label
+(optional)`. The Odoo warehouse name should be entered here, and the characters should be exactly
+the same.
+
+An example of the Sendcloud configuration:
+
+.. image:: sendcloud_shipping/warehouse-label.png
+   :align: center
+   :alt: Warehouse label in the Sendcloud settings.
+
+An example of the Odoo warehouse configuration:
+
+.. image:: sendcloud_shipping/odoo-warehouse.png
+   :align: center
+   :alt: Warehouse settings in Odoo.
```

Comment:
The written context can not rely on imagery to communicate information and needs to be able to stand on it's own; images and screenshots are _supplemental_, only _after_ clear instruction is demonstrated with the written word. 

Creating a symbiotic relationship between the two causes the following quality risks to the documentation:
- visual media leaves too much assumption up to the viewer (especially if they're unfamiliar with the UI);
- prevents users with low internet connections from absorbing the content; and
- opens up more room for "lazy" writing, e.g. "it's easy, just look at the picture!"
- visual media is not searchable/skimmable which disrupts reader's ability to navigate the content quickly

As well, this section would be well suited as an [example block](https://www.odoo.com/documentation/15.0/contributing/documentation/rst_cheat_sheet.html#example) 👍